tests/apps/gt/test_gt_api.py

```diff
@@ -35,7 +35,7 @@ def test_get_trending(genotypeapi, mocker):
     genotypeapi.get_trending(days=days)
 
     # THEN assert subprocess is running the GenotypeAPI with correct command
-    call = f"gtdb --database database prepare-trending -d {days}"
+    call = "gtdb --database database prepare-trending -d %s" % days
```

Comment:
It's really fine to leave the f-string here ;) You do not want to have the f-string in a log message as those only get interpolated if we need them.

The linter would tell you the same thing

Reply:
okej!